lib/registries/addon/branded/index/route.ts

```diff
@@ -0,0 +1,8 @@
+import Route from '@ember/routing/route';
+
+export default class BrandedRegistriesRoute extends Route {
+    beforeModel() {
+        const params: { providerId?: string } = this.paramsFor('branded');
+        return this.replaceWith('branded.discover', params.providerId);
```

Comment:
We should catch the case where someone tries to go to `registries/osf` as well. Per the meeting today, it should go to page-not-found?

Reply:
@handled in on the branded.discover route.